File: src/tool/spring-boot-generator/output/test/db/dealRecord.js
let dealRecord = {
  name: "dealRecord",
  description: "探伤处置记录",
  columns: {
    id: {
      name: "id",
      description: "处置记录编号",
      isPk: true,
      autoIncrement: true,
      nullable: false,
      type: {
        name: "int",
        length: null
      }
    },
    orderDetailId: {
      name: "orderDetailId",
      description: "记录编号",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    workshopInfoId: {
      name: "workshopInfoId",
      description: "处置车间",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    workareaId: {
      name: "workareaId",
      description: "处置人",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    dealTime: {
      name: "dealTime",
      description: "处置时间",
      nullable: true,
      type: {
        name: "datetime"
      }
    },
    dealResult: {
      name: "dealResult",
      description: "处置结果",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    dealRemark: {
      name: "dealRemark",
      description: "处置说明",
      nullable: true,
      type: {
        name: "varchar",
        length: null
      }
    },
    cmdNo: {
      name: "cmdNo",
      description: "施工命令行",
      nullable: true,
      type: {
        name: "varchar",
        length: null
      }
    },
    isDelete: {
      name: "isDelete",
      description: "删除标记",
      nullable: false,
      type: {
        name: "int",
        length: null
      }
    },
    createTime: {
      name: "createTime",
      description: "创建时间",
      nullable: true,
      type: {
        name: "datetime"
      }
    },
    updateTime: {
      name: "updateTime",
      description: "更新时间",
      nullable: true,
      type: {
        name: "datetime"
      }
    },
    operator: {
      name: "operator",
      description: "操作人",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    deleteAuditStatus: {
      name: "deleteAuditStatus",
      description: "删除审计状:-1审计成功,1待审计",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    deleteTime: {
      name: "deleteTime",
      description: "删除时间",
      nullable: true,
      type: {
        name: "datetime"
      }
    }
  }
};

let columnsArray = [
  "id",
  "orderDetailId",
  "workshopInfoId",
  "workareaId",
  "dealTime",
  "dealResult",
  "dealRemark",
  "cmdNo",
  "isDelete",
  "createTime",
  "updateTime",
  "operator",
  "deleteAuditStatus",
  "deleteTime"
];

function create() {
  return {
  id: 597,
  orderDetailId: 173,
  workshopInfoId: 795,
  workareaId: 186,
  dealTime: "2020-1-23 1:3:35",
  dealResult: 498,
  dealRemark: "处置说明224",
  cmdNo: "施工命令行666",
  isDelete: 685,
  createTime: "2019-11-23 1:3:35",
  updateTime: "2019-12-30 1:3:35",
  operator: 605,
  deleteAuditStatus: 686,
  deleteTime: "2019-11-23 1:3:35"
};
}

dealRecord.columnsArray=columnsArray;
dealRecord.create=create;
module.exports=dealRecord;

File: src/tool/spring-boot-generator/output/test/test/src/main/java/com/test/pojo/entity/DealRecord.java
package com.test.pojo.entity;

import com.test.pojo.entity.DealRecord;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;
import javax.validation.constraints.NotNull;



@ApiModel("探伤处置记录")
@Getter
@Setter
public class DealRecord @extends {

    @ApiModelProperty("施工命令行")
    private String cmdNo;

    @ApiModelProperty("创建时间")
    private Date createTime;

    @ApiModelProperty("处置说明")
    private String dealRemark;

    @ApiModelProperty("处置结果")
    private Integer dealResult;

    @ApiModelProperty("处置时间")
    private Date dealTime;

    @ApiModelProperty("删除审计状:-1审计成功,1待审计")
    private Integer deleteAuditStatus;

    @ApiModelProperty("删除时间")
    private Date deleteTime;

    @ApiModelProperty("处置记录编号")
    private Integer id;

    @ApiModelProperty("删除标记")
    @NotNull
    private Integer isDelete;

    @ApiModelProperty("操作人")
    @NotNull
    private Integer operator;

    @ApiModelProperty("记录编号")
    @NotNull
    private Integer orderDetailId;

    @ApiModelProperty("更新时间")
    @NotNull
    private Date updateTime;

    @ApiModelProperty("处置人")
    @NotNull
    private Integer workareaId;

    @ApiModelProperty("处置车间")
    @NotNull
    private Integer workshopInfoId;

}